### PR TITLE
Fixed Inactive students

### PR DESCRIPTION
**Pull request by Atikpui007**
### Summary 

- Show Active students on default
- Show Both Active and Inactive students when box is checked

This pull request is the first step towards implementing feature Foo

- [ ] implemented 'Show Inactive Students checkbox
- [x] fixed default to only show active students

### CODE REVIEW

File: frontend/src/components/UserTables/StudentsTable.tsx [hunk 1]
```diff
@@ -86,18 +86,18 @@ const StudentsTable = ({ searchName }: studentTableProps) => {
         <Row header colSizes={colSizes} data={headers} />
 
         {(filter
-          ? riders.filter(
+          ? riders.filter((r) =>
+              (r.firstName + ' ' + r.lastName)
+                .toLowerCase()
+                .includes((searchName + '').toLowerCase())
```
Comment (Enochen) [inline frontend/src/components/UserTables/StudentsTable.tsx:92]: There's repeated logic here with the filtering, let's refactor that

**Review by Enochen — CHANGES_REQUESTED** [review]

File: frontend/src/components/UserTables/StudentsTable.tsx (hunk 1 above)
Comment (Atikpui007) [reply frontend/src/components/UserTables/StudentsTable.tsx:92]: Got it. Sending a new request to patch this

Comment (Enochen) [reply frontend/src/components/UserTables/StudentsTable.tsx:92]: No need to make a new PR, you can make the changes to the current branch.

File: frontend/src/components/UserTables/StudentsTable.tsx [hunk 2]
```diff
@@ -66,6 +66,15 @@ const StudentsTable = ({ searchName }: studentTableProps) => {
   const formatDate = (date: string): string =>
     moment(date).format('MM/DD/YYYY');
 
+  const filterStudents = (filter: boolean) =>
```
Comment (Enochen) [inline frontend/src/components/UserTables/StudentsTable.tsx:69]: The name `filter` doesn't really describe the function it's providing anymore, now that we're reducing the filtering when the checkbox is on. Let's give it a more appropriate name.

Comment (Enochen) [inline frontend/src/components/UserTables/StudentsTable.tsx:69]: Also, the parameter isn't necessary as we are not going to call this multiple times, and we have everything we need in scope to calculate the result immediately.

ie `const filteredStudents = riders.filter...`

**Review by Enochen — CHANGES_REQUESTED** [review]

File: frontend/src/components/UserTables/StudentsTable.tsx (hunk 2 above)
Comment (Enochen) [reply frontend/src/components/UserTables/StudentsTable.tsx:69]: This comment is talking moreso about the state name from the `useState` call earlier in the file, not this parameter in particular.

Comment (Atikpui007) [reply frontend/src/components/UserTables/StudentsTable.tsx:69]: Got it. I am thinking the the function works the `filter useState `name should be changed to something along the lines of `showInactive` as that is the main purpose of the boolean

Comment (Enochen) [reply frontend/src/components/UserTables/StudentsTable.tsx:69]: Yep, that's a great name

Comment (Atikpui007) [reply frontend/src/components/UserTables/StudentsTable.tsx:69]: Alright. Is there a specific convention that the codebase follows? Arrow functions vs regular functions? Explicit type naming, etc?
eg 
`const filteredStudents = ....` vs `const filteredStudents = () => ....`

**Review by Enochen — APPROVED** [review]
Awesome, thanks for the additional refactoring

**Pull request merged**